gprofiler/profilers/python.py

```diff
@@ -202,6 +202,7 @@ def _make_command(self, pid: int, output_path: str, duration: int) -> List[str]:
         return command
 
     def _profile_process(self, process: Process, duration: int, spawned: bool) -> ProfileData:
+        container_name = self._profiler_state.get_container_name(process.pid)
```

Comment:
Next to appid & app_metadata https://github.com/Granulate/gprofiler/pull/667/files#r1116345068